spyder_kernels/console/start.py

```diff
@@ -269,6 +271,9 @@ def main():
 
     class SpyderKernelApp(IPKernelApp):
 
+        outstream_class = DottedObjectName(
```

Comment:
Please add this constant to the top of the file

```python
IPYKERNEL_6 = ipykernel.__version__[0] >= '6'
```

and here please change the code to be

```python
if IPYKERNEL_6:
    outstream_class = DottedObjectName(
        'spyder_kernels.console.outstream.TTYOutStream')
```

That's because this feature is only available for IPykernel version 6 or greater, so we can't set it for older versions.

Reply:
@ccordoba12 I added the suggestions to the PR. I also updated the base to merge to to `2.x`

Reply:
Thanks!